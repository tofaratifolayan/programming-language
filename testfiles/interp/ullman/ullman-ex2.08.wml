3.0 -# 4.5 +# 6.7 ;
43 / (8 % 3) * 5 ;
