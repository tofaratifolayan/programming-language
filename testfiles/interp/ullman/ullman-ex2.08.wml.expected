5.2
105
